plugins/Over9000/quotes.py

```diff
@@ -0,0 +1,147 @@
+class DBZQuotes:
```

Comment:
This class is unnecessary and can be replaced with a single JSON file to simplify the code

Reply:
Fixed!